share/revng/test/configuration/revng-c/simplify-switch.yml

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) rev.ng Labs Srl. See LICENSE.md for details.
+#
+
+commands:
+  #
+  # Run simplify-switch and test against ground truth
+  #
+  - type: revng-c.test-simplify-switch
+    from:
+      - type: revng-qa.compiled-with-debug-info
+        filter: for-simplify-switch
+    suffix: /
+    command: |-
+      revng artifact --resume "$OUTPUT" decompile-to-single-file --abi-enforcement-level=no --analyses-list=revng-initial-auto-analysis --analyses-list=revng-c-initial-auto-analysis "$INPUT" -o /dev/null;
+      FileCheck --input-file="$OUTPUT/simplify-switch/module.ll" "${SOURCE}.filecheck";
```

Comment:
1. Split this over multiple lines (no `\` at the end of line required)
2. Do not use `--analyses-list=revng-initial-auto-analysis --analyses-list=revng-c-initial-auto-analysis`, just use `--analyze`
3. There's no reason to get to `decompile-to-single-file`. Just do `revng artifact simplify-switch | FileCheck "${SOURCE}.filecheck"`

Reply:
Addressed most of the items (except `3`). Running `revng artifact simplify-switch` on this binary triggers an unrelated assert, so that is why I still use `decompile-to-single-file` (even some other artifacts do not work, e.g. `$ revng artifact --resume workdir/ remove-lifting-artifacts switch-jump-table-revng-qa.compiled-with-debug-info-6f2b090f  -o /dev/null ;` triggers an assert).

Reply:
Please report the exact problem on mattermost, this needs to be fixed.

I doubt that `decompile-to-single-file` succeeds when `simplify-switch` doesn't. However it's not impossible givent that the contract for the pipe is incorrect. Try to fix it as suggested and try again.